frontend/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import {
+  Link
+} from "react-router-dom";
+import { createStyles, Theme, makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import PeopleIcon from "@material-ui/icons/People";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+import AppsIcon from "@material-ui/icons/Apps";
+import AssignmentIcon from "@material-ui/icons/Assignment";
+import EmojiObjectsIcon from "@material-ui/icons/EmojiObjects";
+import SettingsIcon from "@material-ui/icons/Settings";
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
```

Comment:
```
import {
    PeopleIcon,
    NorificationsIcon,
    ...
} from '@material-ui/icons';
```

Reply:
Import in your example doesn't work. I left it like before.